src/api/store/utils/hashing.ts

```diff
@@ -0,0 +1,19 @@
+import { hexlify } from '@ethersproject/bytes';
+import { sign, utils, verify } from 'noble-ed25519';
+
+import type { JsonRPCRequest } from '@types';
+import { stripHexPrefix } from '@utils';
+
+export const hashRequest = async (data: JsonRPCRequest) => {
+  const encoded = Buffer.from(JSON.stringify(data), 'utf-8');
```

Comment:
`JSON.stringify` is not deterministic, so this may lead to false negatives.

Reply:
Any ideas how to solve this?